index.js

```diff
@@ -0,0 +1,67 @@
+const app = new Koa();
+
+Sentry.init({
+  dsn: config.get('sentry.url'),
+  environment: config.get('sentry.environment'),
+});
+
+const server = createServer(app.callback());
+
+const router = Router();
+const generator = new SwaggerAPI();
+generator.addJoiRouter(accountsRouter);
+
+router.get('/api.json', async (ctx) => {
+  ctx.body = JSON.stringify(spec, null, '  ');
+});
+
+app.use(httpLogger());
+app.use(koaPassport.initialize());
+app.use(
+  cors({
+    credentials: true,
+  })
+);
+
+app.use(
+  ignoreAssets(
+    bodyParser({
+      multipart: true,
+      includeUnparsed: true,
+    })
+  )
+);
+
+app.use(helmet());
+
+app.use(errorCatcherMiddleware);
+
+app.use(
+  koaSwagger({
+    routePrefix: '/docs',
+    hideTopbar: true,
+    swaggerOptions: {
+      url: `${config.get('server.baseUrl')}/api.json`,
+    },
+  })
+);
+
+router.get('/', async (ctx) => {
+  ctx.body = 'It works!';
+});
+router.use(accountsRouter.middleware());
+
+app.use(router.middleware());
```

Comment:
Also make you sure you keep the proper project structure. Please add appropriate folders for routers and middlewares